File: java-back-server/src/main/java/space/borisgk/taxi/api/model/dto/UserDto.java
package space.borisgk.taxi.api.model.dto;

import lombok.*;
import space.borisgk.taxi.api.model.entity.AuthServiceData;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import java.util.List;
import java.util.Map;

@Builder
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {
    private String id;

    private String firstName, lastName;
    private String avatarUrl;
    private Map<String, String> socialIds;
}

File: kotlin-back-server/src/main/kotlin/space/borisgk/taxi/api/model/dto/UserDto.kt
package space.borisgk.taxi.api.model.dto

import lombok.*

class UserDto {
    var id: Int? = null
        set(id) {
            field = this.id
        }
    var vkId: Int? = null
        set(vkId) {
            field = this.vkId
        }
    var firstName: String? = null
        set(firstName) {
            field = this.firstName
        }
    var lastName: String? = null
        set(lastName) {
            field = this.lastName
        }
    var photoUrl: String? = null
        set(photoUrl) {
            field = this.photoUrl
        }
}
